integration/spark2/src/test/scala/org/apache/spark/carbondata/TestStreamingTableOperation.scala

```diff
@@ -249,6 +251,195 @@ class TestStreamingTableOperation extends QueryTest with BeforeAndAfterAll {
     assertResult(exceptedRow)(row)
   }
 
+  test("test preaggregate table creation on streaming table without handoff") {
+    val identifier = new TableIdentifier("agg_table", Option("streaming"))
+    val carbonTable = CarbonEnv.getInstance(spark).carbonMetastore.lookupRelation(identifier)(spark)
+      .asInstanceOf[CarbonRelation].metaData.carbonTable
+    val csvDataDir = new File("target/csvdatanew").getCanonicalPath
+    // streaming ingest 10 rows
+    generateCSVDataFile(spark, idStart = 10, rowNums = 5, csvDataDir)
```

Comment:
Is it required to generate csv data for each test case? can't we generate in beforeALL?

Reply:
moved to beforeAll(). In compaction, had to generateCSVDataFile 4 times because checkpoints are maintained